modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx

```diff
@@ -144,19 +144,24 @@ const Table: React.FC<TableProps> = ({
 
 						{columns.map((column, index) => (
 							<ClayTable.Cell headingTitle key={index}>
-								<>
-									{column.value}
+								<div className="d-flex justify-content-between">
+									<span
+										className="cursor-pointer"
```

Comment:
No need to add cursor pointer if the column is not sortable, will give a false impression to the user that he can perform an action.

```suggestion
										className={classNames({'cursor-pointer': column.sortable})}
```